esrally/rally.py

```diff
@@ -143,147 +143,147 @@ def positive_number(v):
                  "e.g. \"@2013-07-27T10:37:00Z\" (default: current).",
             default="current")  # optimized for local usage, don't fetch sources
         p.add_argument(
             "--track",
-            help="define the track to use. List possible tracks with `%s list tracks` (default: geonames)." % PROGRAM_NAME
+            help="Define the track to use. List possible tracks with `%s list tracks` (default: geonames)." % PROGRAM_NAME
             # we set the default value later on because we need to determine whether the user has provided this value.
             # default="geonames"
         )
         p.add_argument(
             "--track-params",
-            help="define a comma-separated list of key:value pairs that are injected verbatim to the track as variables",
+            help="Define a comma-separated list of key:value pairs that are injected verbatim to the track as variables.",
             default=""
         )
         p.add_argument(
             "--challenge",
-            help="define the challenge to use. List possible challenges for tracks with `%s list tracks`" % PROGRAM_NAME)
+            help="Define the challenge to use. List possible challenges for tracks with `%s list tracks`." % PROGRAM_NAME)
         p.add_argument(
             "--team-path",
-            help="define the path to the car and plugin configurations to use.")
+            help="Define the path to the car and plugin configurations to use.")
         p.add_argument(
             "--car",
-            help="define the car to use. List possible cars with `%s list cars` (default: defaults)." % PROGRAM_NAME,
+            help="Define the car to use. List possible cars with `%s list cars` (default: defaults)." % PROGRAM_NAME,
             default="defaults")  # optimized for local usage
         p.add_argument(
             "--car-params",
-            help="define a comma-separated list of key:value pairs that are injected verbatim as variables for the car",
+            help="Define a comma-separated list of key:value pairs that are injected verbatim as variables for the car.",
             default=""
         )
         p.add_argument(
             "--elasticsearch-plugins",
-            help="define the Elasticsearch plugins to install. (default: install no plugins).",
+            help="Define the Elasticsearch plugins to install. (default: install no plugins).",
             default="")
         p.add_argument(
             "--plugin-params",
-            help="define a comma-separated list of key:value pairs that are injected verbatim to all plugins as variables",
+            help="Define a comma-separated list of key:value pairs that are injected verbatim to all plugins as variables.",
             default=""
         )
         p.add_argument(
             "--target-hosts",
-            help="define a comma-separated list of host:port pairs which should be targeted iff using the pipeline 'benchmark-only' "
```

Comment:
fyi: "iff" is not necessarily a typo. It is short-hand for [if and only if](https://en.wikipedia.org/wiki/If_and_only_if) but I admit it's not widely known. However, let's keep your change here.

Reply:
I did not know that, thanks for the explanation @danielmitterdorfer 

Anyway, I feel that for the vast majority of people looking at the docs the fixed spelling is going to look much less suspicious :)